jekyll/_cci2/server/v4.4/operator/circleci-server-security-features.adoc

```diff
@@ -0,0 +1,147 @@
+---
+contentTags:
+  platform:
+  - Server v4.4
+  - Server Admin
+---
+= CircleCI server security features
+:page-layout: classic-docs
+:page-liquid:
+:page-description: This document outlines security features built into CircleCI and related integrations.
+:icons: font
+:toc: macro
+:toc-title:
+
+This document outlines security features built into CircleCI and related integrations.
+
+[#security-overview]
+== Overview
+Security is our top priority at CircleCI. We are proactive and we act on security issues immediately. Report security issues to security@circleci.com with an encrypted message using our security team's GPG key (ID: 0x4013DDA7, fingerprint: 3CD2 A48F 2071 61C0 B9B7 1AE2 6170 15B8 4013 DDA7).
+
+[#encryption]
+== Encryption
+CircleCI uses HTTPS or SSH for all networking in and out of our service, including from the browser to our services application, from the services application to your builder fleet, from our builder fleet to your source control system, and all other points of communication. None of your code or data travels to or from CircleCI without being encrypted, unless you have code in your builds that does so at your discretion. Operators may also choose to bypass our SSL configuration or not use TLS for communicating with underlying systems.
+
+The nature of CircleCI is that our software has access to your code and whatever data that code interacts with. All jobs on CircleCI run in a sandbox (specifically, a Docker container or an ephemeral VM) that stands alone from all other builds and is not accessible from the Internet or from your own network. The build agent pulls code via git over SSH. Your particular test suite or job configurations may call out to external services or integration points within your network, and the response from such calls will be pulled into your jobs and used by your code at your discretion. After a job is complete, the container that ran the job is destroyed and rebuilt. All environment variables are encrypted using link:https://www.vaultproject.io/[Hashicorp Vault]. Environment variables are encrypted using AES256-GCM96 and are unavailable to CircleCI employees.
```

Comment:
I think we need a note here excluding self-hosted runners from this assumption. Their machine may not be ephemeral in that case, and while similar to a docker container runner doesn't run job in a docker container, but rather a container using whichever runtime their cluster is configured to use.